catppuccin-steam.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash 
+set -euo pipefail
+IFS=$'\n\t'
+
+# Install dependencies
+if [[ -f /usr/bin/pacman  && ! -f {/usr/bin/wget,/usr/bin/unzip,/usr/bin/git} ]]
```

Comment:
`command -v` would probably be better here, preferably before running this chain of if statements to see if it's needed in the first place

try to use
```bash
deps=("wget" "unzip" "git")
for dep in "${deps[@]}";
do
  if ! command -v "$dep";
  then
    # all of the package manager stuff
    break;
  fi
done
```